### ISSUE TITLE
redis testing init needs has no system.config at startup

### ISSUE THREAD

**Issue by foxriver76** [ioBroker/ioBroker.js-controller#1143]
- if redis is used for objects testing, the system.config and other default objects are not available during the start, because they are created via `addObjects` in `testAdapter.js`. We should start the controller, call `addObjects` and then stop, this will simulate a the object creation of `setupFirst`

**Comment by foxthefox** [comment]
I observed today that all travis tests are failing, because of the loading of master repo from GitHub. Even old test were looping.
Seems to be in relation to this issue.
`1) Test fritzdect adapter
459       "before all" hook: Test fritzdect adapter: Start js-controller:
460     Uncaught Error: ENOENT: no such file or directory, open '/Users/travis/build/foxthefox/ioBroker.fritzdect/tmp/iobroker-data/objects.json'`

**Comment by foxriver76** [comment]
> I observed today that all travis tests are failing, because of the loading of master repo from GitHub. Even old test were looping.
> Seems to be in relation to this issue.
> `1) Test fritzdect adapter 459 "before all" hook: Test fritzdect adapter: Start js-controller: 460 Uncaught Error: ENOENT: no such file or directory, open '/Users/travis/build/foxthefox/ioBroker.fritzdect/tmp/iobroker-data/objects.json'`

This was another problem which should be fixed now.

**Comment by Apollon77** [comment]
@foxthefox yes we did a bigger js-controlle 3.2 restructuring ... I'm still finalizing it. please try again. pot. clear travis caches

**Comment by foxthefox** [comment]
travis works again as expected, thanks for your efforts
